groupstage.js

```diff
@@ -41,8 +41,14 @@ var makeMatches = function (numPlayers, groupSize, hasAway) {
         }
         else { // players meet twice
           var plsA = plsH.slice().reverse();
-          matches.push({ id: new Id(g+1, mapOdd(r+1), m+1), p: plsH });
-          matches.push({ id: new Id(g+1, mapEven(r+1), m+1), p: plsA });
+          matches.push({ id: new Id(g+1, r+1, m+1), p: plsH });
+          matches.push({ id: new Id(g+1, r+1+rnds.length, m+1), p: plsA });
```

Comment:
the round numbers seem to diverge from the setup in mapEven and mapOdd, is this right?

Reply:
Imagine first match between 1 and 4, then, the match betwen 4 and 1 should be placed when 1 and 4 have played agains 2 and 3. That's the idea, in your code, match 1-4 is followed by match between 4-1 and that is wrong in normal leagues